konspekt_pracy_magisterskiej.md

```diff
@@ -0,0 +1,59 @@
+﻿<![endif]-->
+
+**"Perception of aspiration in English"**
+
+**Abstract**
+
+**Introduction**
+
+Aim of the paper, thesis, introduction to subject, summary of chapters and subchapters.
+
+**Chapter 1 Theory**
+
+1. **Perceptual vs phonetic level**
+In this subchapter I would like to discuss the notion of both phonetic and perceptual level due to the fact that they differ largely i.e. the perception of heard speech sounds may vary as far as listeners are concerned, therefore, I would like to detailedly describe each of these levels. 
+
+2. **Aspiration**
+In this subchapter I would like to discuss the notion of phonological aspiration i.e. explain what it is, give an insight into environment for aspiration and describe the process of it. 
+2.1. **Manner of articulation - stop consonants**
```

Comment:
Please note that your subsections are not rendered as new lines: https://github.com/nlp-ug-2018-19/MA-Thesis-Overview/blob/NikolaGrubecka-patch-2/konspekt_pracy_magisterskiej.md 
Please try to explore the topic.